lamindb/models/artifact.py

```diff
@@ -2146,12 +2146,15 @@ def finalize():
         _track_run_input(self, is_run_input)
         return access
 
-    def load(self, is_run_input: bool | None = None, **kwargs) -> Any:
+    def load(
+        self, mute: bool = False, is_run_input: bool | None = None, **kwargs
+    ) -> Any:
         """Cache and load into memory.
 
         See all :mod:`~lamindb.core.loaders`.
 
         Args:
+            mute: Whether to print the progress of caching.
```

Comment:
Also this should be consistent across the API: "Mute logging." Ideally with one docstring variable that's reused across the codebase and then just injected into the docstring via `@doc_args()`. This will also ensure that we consistently use this pattern when it comes up next time.

Reply:
I would still want to provide more explanations here, like what we are muting or silencing. I have changed to "Silence logging of caching progress".